shell/platform/windows/angle_surface_manager.cc

```diff
@@ -20,25 +20,31 @@ AngleSurfaceManager::~AngleSurfaceManager() {
   CleanUp();
 }
 
-bool AngleSurfaceManager::InitializeEGL(const EGLint* attributes) {
+static void logEglError(std::string message) {
```

Comment:
This isn't tightly bound to a specific method, so I would prefer it be before all other methods. (The desktop embeddings commonly put static helpers at the top like that, often using an anonymous namespace rather than `static` although that's less important.)